src/clientUtils.js

```diff
@@ -44,6 +44,16 @@ client.BaseClient = class {
     });
   }
 
+  handleServerData(data){
```

Comment:
This looks like a default message handler suitable for command-line clients (both for tcp and ws). For 3rd party code, I'd like to make the raw JSON available, so they can handle errors and success their own way in a uniform way, rather than just a raw string.

Can I change this to a separate method that's exported, like
```
client.defaultDataHandler = (data) => {
...
}
```

and then in `scripts/{cli,websock}Client.js` we can add it after instantiating the code:
```
const { defaultMessageHandler, WebSocketClient } = require('..');
const wsc = new WebSocketClient(...);
wsc.addMessageListener(defaultMessageHandler);
```